extras/closeMilestones.js

```diff
@@ -14,6 +14,7 @@ let repos = [
   {owner: 'datahq', repo: 'assembler'},
   {owner: 'datahq', repo: 'pm'},
   {owner: 'datahq', repo: 'specstore'},
+  {owner: 'datahq', repo: 'resolver'},
```

Comment:
This commit would be better as a simple push outside of this pull request (it is not part of the changes described in this PR).

But not a biggie!